tutorials/digital_elevation_models.md

```diff
@@ -0,0 +1,184 @@
+
+  * In the top toolbar, go to Layer > Add Layer > Add XYZ Connection...
+  * In the dropdown, select Google Satellite > Click 'Add' > then 'Close'
+
+* In the top toolbar, go to Layer > Add Layer > Add Raster Layer...
+* Select the file that we downloaded and unzipped (from the instructions above) > Click 'Add' > then 'Close'
+* In the bottom bar, update "Scale" to `1:50000` and in the "Coordinate" textbot, enter the lat/lon coordinates retrieved from USGS earlier (i.e., `37.7444°, -119.5341°`) > hit the Enter key (this will move the viewpoint to the desired location)
+
+  * If the optional step was followed, QGIS should look something like the image below, otherwise only the DEM will be displayed
+
+![QGIS with Google Satellite as base layer and DEM](files/digital_elevation_models/qgis_satellite_and_dem.png)
+
+* If the Processing Toolbox panel isn't open, click the gear icon in the top toolbar
+
+  * Search for "Clip raster by extent" and double click to open the dialog
+  * Input layer: should be the imported geotif
+  * Clipping extent: click the down arrow and select "Draw on Map Canvas" to select your desired region
+
+  ![QGIS draw on canvas](files/digital_elevation_models/qgis_draw_on_map_canvas.png)
+
+  * Clipped (extent): Click the button with the "..." dots > Save to File ... > select the desired save location and give the file a name
+  * Click the "Run" button, this will add a new layer to the layers panel (on the left)
+
+## SDF setup
+
+Now let's set up the SDF model, in a terminal run: `gdalinfo -stats <newly_saved.tif>`
+
+Look for the output: `Size is <x_value> <y_value>`
+
+This is the `x` and `y` values that need to be entered in the `<size>` tag. To calculate z, subtract the `Minimum` from the `Maximum` elevation (i.e., `z = Maximum - Minimum`).
+
+Optionally, to move the model down to Gazebo's camera default viewpoint, update the `z` of the `<pos>` tag to be the negative of the `Maximum` elevation (i.e., `<pos>0 0 -Maximum</pos>`). Note when using seabed data (e.g., GEBCO), negate the `Minimum` instead.
+
+The model should look similar to the following:
+
+```xml
+<model name="half_dome">
+  <static>true</static>
+  <link name="link">
+      <collision name="collision">
+          <geometry>
+              <heightmap>
+                  <uri>half_dome.tif</uri>
+                  <!-- gdalinfo -stats <dem_file> -->
+                  <size>3793 3563 1481.398</size>
+                  <pos>0 0 -2694.987</pos> <!-- -(max or min elevation) -->
+              </heightmap>
+          </geometry>
+      </collision>
+      <visual name="visual">
+          <geometry>
+              <heightmap>
+                  <use_terrain_paging>false</use_terrain_paging>
+                  <texture>
+                      <diffuse>materials/textures/rocks_diffuse.png</diffuse>
+                      <normal>materials/textures/rocks_normal.png</normal>
```

Comment:
looks like these are placed in the `textures` directory (no parent `materials` dir) inside `half_dome_example`. So we probably needs to either:
* update these to `textures/rocks_diffuse.png`, or
* add a parent `materials` directory